### PR TITLE
formulae: run `sudo purge` before `brew style`

### PR DESCRIPTION
**Pull request by carlocab**
We keep seeing `LoadError`s for msgpack, and this seems to make it go
away.

### CODE REVIEW

**Review by Bo98 — APPROVED** [review]
Worth trying to see if it makes the issues disappear.

File: lib/tests/formulae.rb [hunk 1]
```diff
@@ -510,6 +510,11 @@ def formula!(formula_name, args:)
 
         livecheck(formula) if !args.skip_livecheck? && !skip_online_checks
 
+        if ENV["GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED"].present? && OS.mac? && MacOS.version == :sequoia
+          # Fix intermittent broken disk cache on Sonoma after building from source.
```
Comment (ZhongRuoyu) [inline lib/tests/formulae.rb:514]: I've mistaken Sequoia for Sonoma at least 3 times in the past day and it seems I'm not the only one!

Comment (carlocab) [reply lib/tests/formulae.rb:514]: ugh 🤦

Comment (carlocab) [reply lib/tests/formulae.rb:514]: #1212

**Pull request merged**